src/main/java/com/prototype/booking/api/BookResponse.java

```diff
@@ -0,0 +1,10 @@
+package com.prototype.booking.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class BookResponse {
```

Comment:
```suggestion
public class MeetingRoomResponse {
```
Just adhere to what name you will give to existing **BookRoomApp**

Reply:
Removed the Response class, as we can access the reference no from the MeetingRoomBooking entity directly.